packages/dev/loaders/src/OBJ/solidParser.ts

```diff
@@ -537,6 +551,28 @@ export class SolidParser {
         return line.startsWith("g");
     }
 
+    private static _GetZbrushMRGB(line: string, notParse: boolean) {
+        if (!line.startsWith("mrgb")) return null;
+        line = line.replace("mrgb", "").trim();
+        // if include vertex color , not load mrgb anymore
+        if (notParse) return [];
+        const regex = /[a-z0-9]/g;
+        const regArray = line.match(regex);
+        if (!regArray || regArray.length % 8 !== 0) {
+            return null;
+        }
+        const array = [];
+        for (let regIndex = 0; regIndex < regArray.length / 8; regIndex++) {
+            const m = regArray[regIndex * 8 + 0] + regArray[regIndex * 8 + 1];
+            const r = regArray[regIndex * 8 + 2] + regArray[regIndex * 8 + 3];
+            const g = regArray[regIndex * 8 + 4] + regArray[regIndex * 8 + 5];
+            const b = regArray[regIndex * 8 + 6] + regArray[regIndex * 8 + 7];
+            const color = { m: parseInt(m, 16) / 255, r: parseInt(r, 16) / 255, g: parseInt(g, 16) / 255, b: parseInt(b, 16) / 255 };

```

Comment:
Is the `m` property used anywhere?

Reply:
m is not used, m is materialId here, I add it to keep all data save. Should I delete it?